docs/install.md

```diff
@@ -1,28 +1,47 @@
 #Installation and Settings
+##Prerequisites
```

Comment:
There should be one space between the "#", "##"... and the beginning ot the associated heading.